Gemfile

```diff
@@ -25,6 +25,8 @@ end
 
 group :development, :test do
   gem "pry"
+  gem 'pry-stack_explorer'
+  gem 'pry-byebug'
```

Comment:
I'm gonna be that guy and ask you to change the quotes to match the rest of the file.

Reply:
Done!